content/en/docs/concepts/storage/volumes.md

```diff
@@ -1000,469 +727,627 @@ backed by tmpfs (a RAM-backed filesystem) so they are never written to
+Pods interact with FlexVolume drivers through the `flexvolume` in-tree volume plugin.
+For more details, see the [FlexVolume](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-storage/flexvolume.md) examples.
 
-Sometimes, it is useful to share one volume for multiple uses in a single pod.
-The `volumeMounts.subPath` property specifies a sub-path inside the referenced volume
-instead of its root.
+### persistentVolumeClaim {#persistentvolumeclaim}
 
-The following example shows how to configure a Pod with a LAMP stack (Linux Apache MySQL PHP)
-using a single, shared volume. This sample `subPath` configuration is not recommended
-for production use.
+A `persistentVolumeClaim` volume is used to mount a
+[PersistentVolume](/docs/concepts/storage/persistent-volumes/) into a pod. PersistentVolumeClaims are a way for cluster developers to "claim" durable storage (such as a GCE PersistentDisk or an iSCSI volume) without knowing the details of the particular cloud environment.
 
-The PHP application's code and assets map to the volume's `html` folder and
-the MySQL database is stored in the volume's `mysql` folder. For example:
+See the information about [PersistentVolumes](/docs/concepts/storage/persistent-volumes/) for more
+details.
+
+## External storage providers
+
+### awsElasticBlockStore {#awselasticblockstore}
+
+An `awsElasticBlockStore` volume mounts a filesystem stored on an Amazon Web Services (AWS) _[EBS](https://aws.amazon.com/ebs/) volume_ into your pod.  Unlike `emptyDir`, which is erased when a pod is removed, the contents of an EBS
+volume are preserved and the volume is merely unmounted.  This means that an
+EBS volume can be pre-populated with data, and that data can be shared
+between pods.
+
+If you manually specify an `awsElasticBlockStore` volume, you must create an EBS volume using `aws ec2 create-volume` or the AWS API before you can use it.
+
+Dynamic provisioning is a feature of the [AWS EBS CSI driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver). The [AWS EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) provides more information on dynamic provisioning of EBS volumes.
+
+There are some restrictions when using an `awsElasticBlockStore` volume:
+
+* the nodes on which pods are running must be AWS EC2 instances
+* those instances need to be in the same region and availability-zone as the EBS volume
+* EBS only supports a single EC2 instance mounting a volume
+
+#### Creating an AWS EBS volume
+
+Before you can use an EBS volume with a pod, you need to create it.
+
+```shell
+aws ec2 create-volume --availability-zone=eu-west-1a --size=10 --volume-type=gp2
+```
+
+Make sure the zone matches the zone where you want to mount that volume. Also check that the size and EBS volume type are suitable for your use.
+
+#### AWS EBS configuration example
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: my-lamp-site
+  name: test-ebs
```

Comment:
`test-ebs` doesn't feel like a good name for a Pod, to be honest.

Reply:
how about `demo-webserver`?

Reply:
I used "webserver-using-aws-ebs" based on the other comment.